src/main/java/Duke.java

```diff
@@ -1,10 +1,180 @@
+// import libraries here
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static ArrayList<String> list = new ArrayList<>();
+    private static ArrayList<Integer> done = new ArrayList<>();
+    private static ArrayList<String> type = new ArrayList<>();
+    private static ArrayList<String> date = new ArrayList<>();
+
+    private static void greeting() {
+        System.out.println("    ____________________________________________________________");
```

Comment:
Since this is used multiple times in the code, i feel is better to create a constant DIVIDER or something.